lib/resources/dstu2/condition.yaml

```diff
@@ -71,6 +72,15 @@ fields:
                    "display": "Generalized nonconvulsive epilepsy",
                    "userSelected": true}],
                 "text": "Absence seizures"}}
+  binding:
+    description: A code that defines the condition.
+    terminology:
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+    - display: ICD-9
+      system: http://hl7.org/fhir/sid/icd-9
```

Comment:
Is this the correct system for ICD-9?  The ICD-10 system url redirects to a Terminologies-systems page which lists all of the valid systems including that specific ICD-10 system, however it doesn't list this specific ICD-9 system. Instead it lists http://hl7.org/fhir/sid/icd-9-cm.  'http://hl7.org/fhir/sid/icd-9' does link to a FHIR page, but it looks like it is a working/temporary page. The page title literally has 'C:\work\org.hl7.fhir\...' in it.